### PR TITLE
SPRT-146: Restoring the previous state.

### PR DESCRIPTION
**Pull request by PabloBerrettoni**
Fixing some pagination problems suggested by @olea on this issue: [#27](https://github.com/PukaraES/Cicerone/issues/27)
When coming back from the detail view of the monuments you'll find the previos pagination and/or the previous location selected

### CODE REVIEW

**Review by njrzr — CHANGES_REQUESTED** [review]
@PabloBerrettoni checking the site I saw some issues, the first one is that when returning the miniatures the select is on the default state (monuments) instead of the selected one (I don't know if a priority to fix it), it should be the same when entered on the details miniature, the second one is when searching for another place and select it, in the map a marker icon is in place where the people icon is, but the people icon is underneath it, I don't know is this was added in this change or another, ask @facuzero for help, to change this and hide the people icon when this happens.

File: components/WLCloseByMiniatures.vue [hunk 1]
```diff
@@ -354,6 +388,7 @@ export default {
       this.$store.commit('setLatitude', this.coords["P625"][0].mainsnak.datavalue.value.latitude)
       // eslint-disable-next-line
       this.$store.commit('setLongitude', this.coords["P625"][0].mainsnak.datavalue.value.longitude)
+      console.log(this.$store.state)
```
[the remark this answers is not in the document]
Comment (PabloBerrettoni) [reply components/WLCloseByMiniatures.vue]: @njrzr This was already fixed [here](https://github.com/PukaraES/Cicerone/pull/84/commits/1dca8312ac53566215fdeb4833127263627ef1d9)

**Comment by PabloBerrettoni** [comment]
> @PabloBerrettoni checking the site I saw some issues, the first one is that when returning the miniatures the select is on the default state (monuments) instead of the selected one (I don't know if a priority to fix it), it should be the same when entered on the details miniature, the second one is when searching for another place and select it, in the map a marker icon is in place where the people icon is, but the people icon is underneath it, I don't know is this was added in this change or another, ask @facuzero for help, to change this and hide the people icon when this happens.

The second issue (the one about the marker) is an already existing issue. Maybe we can create another github issue to address that? Attaching image of that happening on cicerone.guide
image

**Comment by gnaomi0** [comment]
@PabloBerrettoni this is working for me. @njrzr Please check. Also, Pablo I think you can open a new issue about the marker thing.

**Review by njrzr — APPROVED** [review]

**Pull request merged**